west.yml

```diff
@@ -72,6 +72,12 @@ manifest:
       path: modules/hal/gigadevice
       groups:
         - hal
+    - name: hal_hpmicro
+      url: https://github.com/hpmicro/zephyr_hal_hpmicro.git
```

Comment:
this HAL comes with code not needed in Zephyr. Why do we need to pull that content? HAL modules are already a mess, let's not make it worse.

ref https://github.com/hpmicro/zephyr_hal_hpmicro/tree/manifest-rev/hpm_sdk/boards/hpm6360evk

Reply:
The entire `board` folder has been deleted